src/main/java/graphql/execution/instrumentation/NoOpInstrumentation.java

```diff
@@ -1,24 +1,32 @@
 package graphql.execution.instrumentation;
 
 import graphql.ExecutionResult;
+import graphql.PublicApi;
 import graphql.execution.instrumentation.parameters.InstrumentationDataFetchParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters;
+import graphql.execution.instrumentation.parameters.InstrumentationExecutionStrategyParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationFieldParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationValidationParameters;
 import graphql.language.Document;
 import graphql.validation.ValidationError;
 
 import java.util.List;
+import java.util.concurrent.CompletableFuture;
 
 /**
- * Nothing to see or do here ;)
```

Comment:
This makes it easier to create Instrumentation instances because you only have to override the methods you want